scss/_navbar.scss

```diff
@@ -32,6 +32,31 @@
     align-items: center;
     justify-content: space-between;
   }
+
+  .dropdown-menu {
+    position: static;
+    float: none;
+    margin-top: 0;
+    border: 0;
+    border-radius: 0;
```

Comment:
@zalog sorry, I missed 2 this:
First:
You are removing `border` and `border-radius` when the drop menu is within a navbar always, even in large view ports. Remember that border radius is an opt in and we should keep them if the user chooses to do it.